python/pyspark/sql/tests.py

```diff
@@ -1036,8 +1036,15 @@ def test_access_column(self):
         self.assertRaises(TypeError, lambda: df[{}])
 
     def test_column_name_with_non_ascii(self):
-        df = self.spark.createDataFrame([(1,)], ["数量"])
-        self.assertEqual(StructType([StructField("数量", LongType(), True)]), df.schema)
+        if sys.version >= '3':
```

Comment:
Is this the only place we use unicode() in pyspark? this otherwise looks like something that might need to be applied to other tests that try something similar?